src/algorithms/digi/PhotoMultiplierHitDigi.h

```diff
@@ -28,15 +28,20 @@
 
 namespace eicrecon {
 
+struct PhotoMultiplierHitDigiResult {
+  std::unique_ptr<edm4eic::RawTrackerHitCollection> raw_hits;
+  std::unique_ptr<edm4eic::MCRecoTrackerHitAssociationCollection> hit_assocs;
+};
```

Comment:
This is now not necessary anymore thanks to multifactories which allow a simple return value of `std::tuple<edm4eic::RawTrackerHitCollection, edm4eic::MCRecoTrackerHitAssociationCollection>`, and you can `return {raw_hits, hit_assocs}`. As applied here: https://github.com/eic/EICrecon/blob/main/src/algorithms/tracking/ParticlesFromTrackFit.cc#L140 (with probably unnecessary move semantics).

Reply:
One could of course argue whether a struct or pair are really all that different.

Reply:
I'm aware of this trend, but I prefer a struct or class, since:
- easier to find what is included in the output when reading through `AlgorithmProcess`: just search for the returned struct instead of each of the tuple members
- easier to use from the caller, e.g., to get the raw hits, say `result.raw_hits` instead `result->first`; however, it is easier to loop over a tuple or pair, which can be convenient in factories

My preference isn't too strong, so if you'd rather this algorithm be more consistent with others, I'll switch it to a tuple.